package.json

```diff
@@ -21,41 +21,42 @@
   "dependencies": {
     "@alrra/travis-scripts": "3.0.1",
     "@google-cloud/storage": "1.6.0",
+    "@octokit/rest": "15.9.5",
     "ajv": "5",
     "babel-core": "6.26.3",
     "babel-eslint": "8.2.3",
     "babel-loader": "7.1.4",
     "babel-plugin-react-intl": "2.3.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-flow-strip-types": "6.22.0",
-    "babel-preset-env": "1.6.1",
+    "babel-preset-env": "1.7.0",
     "babel-preset-flow": "6.23.0",
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-2": "6.24.1",
     "chalk": "2.4.1",
     "chromeless": "1.4.0",
     "convict": "4.0.1",
-    "coveralls": "2.13.1",
-    "css-loader": "0.28.11",
+    "coveralls": "3.0.2",
+    "css-loader": "1.0.0",
     "eslint": "4.19.1",
     "eslint-config-prettier": "2.9.0",
     "eslint-plugin-flowtype": "2.40.1",
     "eslint-plugin-react": "7.5.1",
     "file-loader": "1.1.11",
     "gettext-parser": "1.2.2",
-    "github": "12.0.1",
     "glob": "7.1.2",
     "googleapis": "23.0.0",
     "gunzip-maybe": "1.4.1",
     "isomorphic-style-loader": "4.0.0",
     "mkdirp": "0.5.1",
     "node-insights": "1.0.16",
-    "node-sass": "4.9.0",
+    "node-sass": "4.9.3",
     "postcss-css-variables": "0.9.0",
-    "postcss-cssnext": "3.1.0",
     "postcss-loader": "2.1.4",
+    "postcss-preset-env": "5.3.0",
     "prettier": "1.12.1",
     "raw-loader": "0.5.1",
+    "react": "16.3.1",
```

Comment:
A required peer dep of `react-hot-loader`, even though we don't directly use it.

Reply:
could this be defined as a peerDependency?

Reply:
yeah - fine to leave the new dependency in for now. I think at some point we're going to want to create some sort of 'master package' that includes all the dependencies we need _once_, but that's for later on.

Since it's an unused dependency, you might want to do something like `"react": "^16.3"` to reduce the need to update for point versions